dispatcher/subscription.go

```diff
@@ -127,6 +131,15 @@ func (sm *SubscriptionMapper) matchLevelFromInput(subEntry data.SubscriptionEntr
 	return MatchAll
 }
 
+func (sm *SubscriptionMapper) getEventType(subEntry data.SubscriptionEntry) string {
```

Comment:
this function doesn't have to belong to `SubscriptionMapper` as it doesn't use anything from it

Reply:
In case we decide to make this public, it is always preferred to have it attached to a struct as to use our friend, polymorphism.

Reply:
set method as a separate private function